### PR TITLE
Change log level for a few items from SEVERE to WARN.

### PR DESCRIPTION
**Pull request by mediocretes**
SEVERE should be reserved for things that will absolutely 100% prevent proper agent functioning, not retryable items or malformed metrics in an otherwise correctly configured agent.

### CODE REVIEW

File: src/main/java/com/instrumentalapp/Connection.java [hunk 1]
```diff
@@ -256,7 +256,7 @@ public void setShutdown(boolean shutdown) {
 				try {
 					worker.join(10000);
 				} catch (InterruptedException ie) {
-					LOG.severe("Failed to cleanly shutdown the Connection");
+					LOG.warning("Failed to cleanly shutdown the Connection");
```
Comment (janxious) [inline src/main/java/com/instrumentalapp/Connection.java:259]: I agree this isn't probably severe, but is there an actual bad thing that will happen in this instance?

File: src/main/java/com/instrumentalapp/Connection.java [hunk 2]
```diff
@@ -273,7 +273,7 @@ private void backoffReconnect() throws InterruptedException {
 		// Do a reconnect, then try to send the message again.
 		// Since we've removed the message from the queue, don't discard it, and don't advance.
 		long delay = (long) Math.min(maxReconnectDelay, Math.pow(errors++, reconnectBackoff) * 1000);
-		LOG.severe("Connection failed to " + agentOptions.getHost() + ":" + agentOptions.getPort() + ". Retry in " + delay + "ms");
+		LOG.warning("Connection failed to " + agentOptions.getHost() + ":" + agentOptions.getPort() + ". Retry in " + delay + "ms");
```
Comment (janxious) [inline src/main/java/com/instrumentalapp/Connection.java:276]: Generally speaking we should be using string formatting instaed of string concatenation, but that's like a problem for future us.

**Review by janxious — APPROVED** [review]
DNRT

File: src/main/java/com/instrumentalapp/Connection.java (hunk 1 above)
Comment (mediocretes) [reply src/main/java/com/instrumentalapp/Connection.java:259]: At this point you are already exiting, so the bad thing is that you may miss some metrics that were enqueued but not yet sent.  This is the one I was least sure about.

File: src/main/java/com/instrumentalapp/Connection.java (hunk 2 above)
Comment (mediocretes) [reply src/main/java/com/instrumentalapp/Connection.java:276]: Java!

File: src/main/java/com/instrumentalapp/Connection.java (hunk 1 above)
Comment (janxious) [reply src/main/java/com/instrumentalapp/Connection.java:259]: Yeah I was more concerned about dangling sockets or something.

File: src/main/java/com/instrumentalapp/Connection.java (hunk 2 above)
Comment (janxious) [reply src/main/java/com/instrumentalapp/Connection.java:276]: ☕️

**Pull request merged**